### ISSUE TITLE
is this part of the README out of date? "This library will change dramatically with the release of React 0.13"

### ISSUE THREAD

**Issue by jamesmanning** [Asana/typed-react#54]
Since the current React version is 0.13.3 and the 0.14 alpha's have started, I wasn't sure if the "This library will change dramatically with the release of React 0.13" part of the README was just outdated or whether typed-react didn't support 0.13 yet.

Thanks!

**Comment by pspeter3** [comment]
Basically, with React 0.13, you don't need TypedReact. it is still compatibly however

**Comment by flyon** [comment]
yes please, if you have any experience with React 0.13 + typescript already, update the README to say that you shouldn't need TypedReact (perhaps with a short explaination how it can work without)
I came here to see how I can use React 0.13 together with typescript and if I need to use Asana's definitions/typed react

**Comment by pspeter3** [comment]
@flyon what do you mean? We support definitions through DefinitelyTyped.

**Comment by daxfohl** [comment]
@pspeter3 When you google "typescript reactjs", this is where you land.  So, as someone who hasn't used typescript and wants to try it out on a react project, I look at the readme and think, "oh I'll wait until this update before starting the project".  And then I come back a few days later and the comment is still there, and am left wondering when it'll be ready.  So I end up having the same question as James.

But, going by your comment above, I'm understanding that with react 0.13, we should actually just pull the [DefinitelyTyped definitions](https://github.com/borisyankov/DefinitelyTyped/tree/master/react) (which doesn't even appear on the first page of the google search), and ignore this repository altogether.  Is that right?

If that's the case, then does this repo serve any purpose going forward?

**Comment by pspeter3** [comment]
It sounds like, the README needs to be updated. I will do that now and appreciate any feedback.

**Comment by flyon** [comment]
Yes, sorry, that's all I meant to. To update the README with some note that if you're using react 0.13 you can just use the definitions from DefinitelyTyped

**Comment by jamesmanning** [comment]
whoops, just missed this - I had made an attempted edit with https://github.com/Asana/typed-react/pull/55 but if you're going to edit it anyway, just ignore that PR :)

**Comment by pspeter3** [comment]
Thoughts? https://github.com/Asana/typed-react/commit/9008be2f41c56bdd975213a08c97f1847333b15d

**Comment by jamesmanning** [comment]
Looks great to me!
